examples/ensemble/plot_forest_importances_faces.py

```diff
@@ -23,8 +23,7 @@
 
 # Load the faces dataset
 data = fetch_olivetti_faces()
-X = data.images.reshape((len(data.images), -1))
-y = data.target
+X, y = data.data, data.target
```

Comment:
why not using return_X_y here

Reply:
because we are using `data.images` later in the example.